RELEASE_NOTES.md

```diff
@@ -3,6 +3,28 @@ Release notes
 
 ### Next
 
+* Add support for the Address History user account creation attribute
+* Add ENVIRONMENT configuration option to replace hubSsoLocation and metadataUrl.
+* Send version number to hub
+* Support multitenancy
+* Improve documentation
+* Improve healthcheck logging
+
+#### Configuration Changes:
+When using environment variables:
+* Replace SERVICE_ENTITY_ID with SERVICE_ENTITY_IDS, which is a JSON string array containing the entity id of the service (or services) using the Verify Service Provider (e.g. '["http://entity-id"]')
```

Comment:
Should probably mention that these block capital things are environment variables?